compiler/noirc_frontend/src/elaborator/statements.rs

```diff
@@ -435,7 +437,19 @@ impl<'context> Elaborator<'context> {
         None
     }
 
-    pub(super) fn elaborate_comptime(&self, _statement: Statement) -> (HirStatement, Type) {
-        todo!("Comptime scanning")
+    pub(super) fn elaborate_comptime(&mut self, statement: Statement) -> (HirStatement, Type) {
+        let span = statement.span;
```

Comment:
```suggestion
    fn elaborate_comptime(&mut self, statement: Statement) -> (HirStatement, Type) {
```

Reply:
`span` is used in `push_expr_location` below after `statement` is consumed on the next line so I figured I'd copy it here instead of retrieving it from the NodeInterner later